TrafficCapture/dockerSolution/src/main/docker/elasticsearchTestConsole/Dockerfile

```diff
@@ -64,7 +64,8 @@ RUN dnf install -y \
         python3.11-pip \
         tar \
         unzip \
-        vim-minimal \
+        vim \
+        which \
```

Comment:
This one was weird to me.  Why doesn't the shell already have which?  I'm used to that being a built-in command.